src/clojure_tictactoe/game/board.clj

```diff
@@ -0,0 +1,11 @@
+(ns clojure-tictactoe.game.board)
+
+(defn open?
+  [space board]
+  (int? (board space)))
+
+
+(defn mark-space
+  [space marker board]
+  (when (open? space board)
```

Comment:
Why not use a conditional `if` statement here?

Reply:
"If you only want to do one thing when your test is true, and don't really care to do anything when it is false, you can use a `when`" - Living Clojure

^ is my reasoning. Happy to hear the reasoning for using `if` instead.

Reply:
Cool 👍